test/functional/apps/dashboard/_dashboard_clone.js

```diff
@@ -45,9 +45,11 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.common.clickCancelOnModal();
       await PageObjects.dashboard.confirmClone();
 
-      // Should see the same confirmation if the title is the same.
-      const isConfirmOpen = await PageObjects.common.isConfirmModalOpen();
-      expect(isConfirmOpen).to.equal(true);
+      await retry.try(async () => {
```

Comment:
Do you know if retry.try is fulfilled with the result of the callback?  If so, couldn't this sort of thing be written as:

```js
const isConfirmOpen = await retry.try(PageObjects.common.isConfirmModalOpen);
expect(isConfirmOpen).to.equal(true);
```

No need to change it or anything, I was just curious.

Reply:
`retry.try` does resolve to the resolution value of the first successful attempt, but I specifically want to retry the assertion and I also don't think we should be breaking the `this` context of `PageObjects.common.isConfirmModalOpen`